server/sonar-server/src/main/java/org/sonar/server/rule/RubyRuleService.java

```diff
@@ -50,11 +50,13 @@
 @ServerSide
 public class RubyRuleService implements Startable {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)